lib.sh

```diff
@@ -947,7 +983,7 @@ done
 }
 
 print_text_in_color() {
-	printf "%b%s%b\n" "$1" "$2" "$Color_Off"
+printf "%b%s%b\n" "$1" "$2" "$Color_Off"
```

Comment:
If you want to to indentation on other stuff at the same time, sure! Why not. :)

Many devs, many styles. I have tried to keep it clean, but as you said, there's no structure in some of it. 
```
function() {
[should start at the first line, or what do you think? Intended 4 spaces here as well?
}
```
```
if arguments
    [as I posted before]
fi
```

Reply:
In a few functions, the first code in function has  indention，but most of them have no indention.

I approve that the coding style should be consistent with the majority of the code.